src/Stripe/Services/Customers/StripeCustomerService.cs

```diff
@@ -48,6 +49,12 @@ public virtual IEnumerable<StripeCustomer> List(StripeCustomerListOptions listOp
             );
         }
 
+        public virtual StripeDiscountDelete DeleteDiscount(string customerId, StripeRequestOptions requestOptions = null) 
+        {
+            var customerUrl = string.Format(Urls.CustomerDiscount, customerId);
+            return Mapper<StripeDiscountDelete>.MapFromJson(Requestor.Delete(customerUrl, SetupRequestOptions(requestOptions)));
+        }
+#if !PORTABLE
```

Comment:
Notice I am commenting ON THIS LINE - that means (for GitHub) - that I am talking about line 57 here. You can remove this  (#if !PORTABLE .... #endif) - those preprocessor statements are no longer needed since async works on the portable version as well. 

let me know if my comments come across as condenscending- I intend for them to help, not be an a**hole. :)